go.mod

```diff
@@ -5,16 +5,16 @@ go 1.21
 require (
 	github.com/blang/semver/v4 v4.0.0
 	github.com/onsi/ginkgo/v2 v2.17.1
-	github.com/onsi/gomega v1.30.0
+	github.com/onsi/gomega v1.31.1
 	github.com/openshift/library-go v0.0.0-20240116081341-964bcb3f545c
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.64.1
 	github.com/spf13/cobra v1.8.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stolostron/kubernetes-dependency-watches v0.5.2
-	k8s.io/apimachinery v0.29.1
-	k8s.io/client-go v0.29.1
+	k8s.io/apimachinery v0.29.2
+	k8s.io/client-go v0.29.2
 	k8s.io/component-base v0.29.1
-	k8s.io/klog/v2 v2.110.1
+	k8s.io/klog/v2 v2.120.1
 	open-cluster-management.io/addon-framework v0.9.1-0.20240321030609-bffb77d5d895
```

Comment:
Could you please update the addon-framework to 0.9.2?

Reply:
https://github.com/open-cluster-management-io/governance-policy-addon-controller/pull/150